web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1334,6 +1334,13 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
     }
   }
 
+  /**
+    * The root path for assets folder. Assets under this configurable path
+    * follow conventions, for example the default lazy loading spinner
+    * ('ajax-loader.gif') is under 'images' folder.
+    */
+  @volatile var assetPath: String = "/assets"
```

Comment:
I'm kind of torn over whether or not this default represents a breaking change or not. I'm inclined to say that we should default this to `/` until Lift 4.0.

Open to other opinions, though.

Reply:
You're right, it would be a nasty surprise when upgrading to a new version of lift. I'll update the PR soon with the default path set to /. Thanks.